internal/assets/manifests/istio-meshgateway/templates/deployment.yaml

```
@@ -51,6 +51,9 @@ spec:
             - containerPort: 15090
               protocol: TCP
               name: http-envoy-prom
+            - containerPort: 15020
+              protocol: TCP
+              name: status
```

Can we rename this to have `http-` prefix to align with Istio port naming conventions?
```suggestion
              name: http-status
```

---

Yes, you are correct, I renamed it to the health check port and I also added the combined metrics port to be exposed in the next commit.